README.md

```diff
@@ -1,36 +1,35 @@
 This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://nextjs.org/docs/app/api-reference/cli/create-next-app).
 
-## Getting Started
+# Enriconsolas
 
-First, run the development server:
+## 1. 🎯 Objetivo del proyecto
 
-```bash
-npm run dev
-# or
-yarn dev
-# or
-pnpm dev
-# or
-bun dev
-```
+El objetivo de este proyecto es el de realizar un marketplace de usuarios donde puedan comprar y vender consolas y videojuegos de segunda mano y darles una segunda vida.
+Un rasgo principal de la aplicación va a ser la simplicidad de la misma. Quiero que un usuario pueda comprar un producto en menos de 10 clicks.
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+## 2. 👥 Actores
+Lista de actores que interactúan con el sistema:
+- **Comprador**: Persona que utiliza la plataforma con intención de comprar consolas o videojuegos.
+- **Vendedor**: Persona que utiliza la plataforma con intención de vender consolas o videojuegos.
 
-You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
+## 3. 🚀 Casos de uso
+* [CF-001 - Listar oferta](./docs/usecases/CF-001.md)
+* 
 
-This project uses [`next/font`](https://nextjs.org/docs/app/building-your-application/optimizing/fonts) to automatically optimize and load [Geist](https://vercel.com/font), a new font family for Vercel.
+## 4. 🪜 Alcance del proyecto
 
-## Learn More
+Delimitar qué se va a incluir en esta versión del proyecto y qué se deja fuera (por tiempo o complejidad). Esto evita que el alcance se dispare y da claridad al evaluador.
 
-To learn more about Next.js, take a look at the following resources:
+## 5. 🧩 Entidades del dominio
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+Pequeño esquema o diagrama de relaciones (tipo ER) entre las entidades para que sea más visual y fácil de entender.
 
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js) - your feedback and contributions are welcome!
+## 6. 📐 Convenciones
 
-## Deploy on Vercel
+Convenciones que se van a seguir (nomenclatura de archivos, variables, estilos).
 
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+## 7. 🏗️ Arquitectura y Tecnologías
 
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/app/building-your-application/deploying) for more details.
+Lista de referencias de las distintas tecnologías y patrones arquitectónicos usados en el proyecto.
+
+* React Server Side Rendering
```

Comment:
Also, don't forget to expand these sections with the stuff we've been talking up to now: DDD, Conventional Commits, Git Flow...